src/main/java/de/tum/cit/aet/artemis/core/web/AccountResource.java

```diff
@@ -133,46 +137,87 @@ public ResponseEntity<Void> changePassword(@RequestBody PasswordChangeDTO passwo
     }
 
     /**
-     * PUT account/ssh-public-key : sets the ssh public key
+     * GET account/ssh-public-keys : sets the ssh public key
+     *
+     * @return the ResponseEntity containing all public SSH keys of a user with status 200 (OK), or with status 400 (Bad Request)
+     */
+    @GetMapping("account/ssh-public-keys")
+    @EnforceAtLeastStudent
+    public ResponseEntity<List<UserSshPublicKey>> getSshPublicKey() {
```

Comment:
`UserSshPublicKey` is a database entity rather than a DTO. This should not be used for new endpoints.